runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/external/ExternalPythonExecutor.java

```diff
@@ -100,7 +100,7 @@ public PythonEvaluationResult eval(String expression, String prepareEnvironmentS
         try {
             String pythonPath = checkPythonPath();
             tempEvalEnvironment = generateTempResourcesForEval();
-            String payload = generatePayloadForEval(expression, prepareEnvironmentScript, context);
+            String payload = generatePayloadForEval(expression.replaceAll("[\\n\\r]", ""), prepareEnvironmentScript, context);
```

Comment:
is this consistent? if the user is allowed in first place to write multi line expression, he will expect that multiline to work. If we replace the newline with blank/spaces, we will break the expression. (considering there are two statements, on separate lines)

Reply:
@alexbajzat a valid expression should be resolved to a pyhton value and cannot have statements, we should replace with space and we should not break expression.

Reply:
Python has native support of separating a statement into a multiline one by simply using the \ character.
Additionally, you can also write multiple statements on one line if you separate them with semicolons.
This change feels like an unnecessary hack to me.

Reply:
Yes, in the case when you are using that \ for splitting the expression into multiple lines it will work.
But we agreed that we want to let user to also write expression in multiple lines without \ (which is not supported in python).
This change is specially for cases like that.

Reply:
+ in case of a multi-line string inside an expression (written with triple quotes), it will remove the completely valid \n characters and turn the user's string into a single line string at runtime

Reply:
I agree, but how should we support then multiple lines expressions?

Reply:
@alexcampanu nice catch!

Reply:
@ramona-grad you could use the tree in client to determine which tokens are strings and replace newlines only in non-string tokens.